actors/common/src/test/java/org/sunbird/learner/actors/BackgroundServiceActorTest.java

```diff
@@ -45,7 +44,11 @@ public static void setUp() {
 
   @Test
   public void updateUserCountTest() {
-
+    Map<String, Object> locnMap = new HashMap<String, Object>();
+    locnMap.put(JsonKey.ID, locnId);
+    locnMap.put(JsonKey.USER_COUNT, 0);
+    cassandraOperation.updateRecord(
```

Comment:
This testcase will not work , if you cassandra is not up & running.
Its better to testcases using powermock